### PR TITLE
enabling grpc commands to actuate motor

### PR DESCRIPTION
**Pull request by bwz5**

### CODE REVIEW

**Comment by bwz5** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/135?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#135** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/135?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#128** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/128?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#122** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/122?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#84** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/84?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#83** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/83?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bwz5 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: fill/fill_station.cc [hunk 1]
```diff
@@ -126,34 +130,15 @@ MAIN
...
-<<<<<<< HEAD
-    // absl::ParseCommandLine(argc, argv);
-    // // Start the server in another thread
-    // std::shared_ptr<Server> server;
-    // RunServer(absl::GetFlag(FLAGS_server_port), server);
-    // // std::thread serverThread(RunServer, absl::GetFlag(FLAGS_server_port), server);
-=======
-    // begin the ADC
-    adc.begin();
+    WiringPiSetup();
```
Comment (ZachGarcia42) [inline fill/fill_station.cc:133]: Move this to the stepper motor object constructor

File: fill/fill_station.cc [hunk 2]
```diff
@@ -71,6 +72,9 @@ class CommanderServiceImpl final : public Commander::Service
     Status SendCommand(ServerContext *context, const Command *request,
                        CommandReply *reply) override
     {
+        if (request->qd_retract) {
```
Comment (ZachGarcia42) [inline fill/fill_station.cc:75]: Should be request->qd_retract() since it's a function call

**Review by ZachGarcia42 — CHANGES_REQUESTED** [review]
A few bugs found during end to end test

File: fill/fill_station.cc (hunk 1 above)
Comment (ZachGarcia42) [inline fill/fill_station.cc:133]: Also I figured out that WiringPiSetupGpio() uses the standard gpio pin numbers which is why we were getting issues. Please update this to use WiringPiSetupGpio() and update the stepper motor pins to the correct ones (GPIO numbering) on the Pi Hat confluence page

**Review by ZachGarcia42 — APPROVED** [review]
Looks Great! Thanks for the fixes Ben!

**Comment by bwz5** [comment]
### Merge activity

* **Nov 6, 11:28 PM EST**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/135).

**Pull request merged**